www-client/otter-browser/otter_browser-0.9.12.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A lightweight QtWebKit browser"
+DESCRIPTION="Project aiming to recreate the best aspects of the \
+classic Opera (12.x) UI using Qt5."
+HOMEPAGE="http://otter-browser.org/"
+COPYRIGHT="2013-2017 Otter Browser Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/OtterBrowser/otter-browser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="1a184561b9dbf3443cf1f83ad8604976b1b06cc00a93306331b364f237363e20"
+ADDITIONAL_FILES="otter.rdef.in"
+SOURCE_DIR="otter-browser-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	otter_browser$secondaryArchSuffix = $portVersion
+	app:OtterBrowser$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libhunspell-1.4$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5DBus$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Multimedia$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Qml$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5XmlPatterns$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake$secondaryArchSuffix
+	cmd:make
+	cmd:g++$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libhunspell-1.4$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5DBus$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Multimedia$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Qml$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5XmlPatterns$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	cmake .. -DCMAKE_INSTALL_PREFIX:PATH="$appsDir/Otter\ Browser"
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+
+	mv $appsDir/Otter-Browser/bin/otter-browser \
+		$appsDir/Otter-Browser/Otter-Browser
```

Comment:
nope, it is defined so in cmakelist.txt, i don't want to use advanced sed magic for this, sorry.